Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM ubuntu:18.04 as build
+FROM ubuntu:22.04 as build
 
 RUN apt-get update && \
     apt-get install -y \
         binutils-mips-linux-gnu \
```

Comment:
anyway yeah add `gcc-mips-linux-gnu` here and it should be good to go

Reply:
Isn't that part of binutils-mips-linux-gnu?

Reply:
they are packaged separately on ubuntu

Reply:
Is there a reason why that particular gcc is used? I've built other SM64 decomp forks fine without it.

Reply:
Repositories based off n64decomp/sm64 are using the original sm64 compiler from 1994, which is included in the repo and thus only needs the regular gcc package and the mips binutils

This repo is based off CrashOveride95/ultrasm64, which among other things makes modern mips gcc the only compiler option for the game, so that package gets added onto the dependency list